R/lightgbm.R

```diff
@@ -238,6 +242,18 @@ train_lightgbm <- function(x,
     free_raw_data = free_raw_data
   )
 
+  if (save_tree_error) {
+    print("Enabling save_tree_error")
```

Comment:
I found these print statements to be useful as I was debugging training behavior, but I'm happy to remove it if you're worried about cluttering up the logs. Or perhaps we only print it if `verbose = TRUE`? I'm still not sure what the idiomatic way to do log levels is in R.

Reply:
**suggestion:** I would do prints only if `verbose > 0` and replace the `print()` calls with `message()`.

Reply:
Good call, done in [8940acc](https://github.com/ccao-data/lightsnip/pull/10/commits/8940acc93fc1bdd5afb23210ae93789adbb25613)!